app/views/spotlight/shared/_curation_sidebar.html.erb

```diff
@@ -6,14 +6,13 @@
   <h4><%=t(:'spotlight.curation.sidebar.header') %></h4>
   <ul class="nav sidenav">
     <li><%= link_to t(:'spotlight.curation.sidebar.items'), spotlight.admin_exhibit_catalog_index_path(@exhibit) %></li>
-    <% if can? :update, @exhibit.blacklight_configuration %>
-    <li><%= link_to t(:'spotlight.curation.sidebar.metadata'), spotlight.exhibit_edit_metadata_path(@exhibit) %></li>
-    <% end %>
     <% if can? :tag, @exhibit %>
     <li><%= link_to t(:'spotlight.curation.sidebar.tags'), spotlight.exhibit_tags_path(@exhibit) %></li>
     <% end %>
     <% if can? :update, @exhibit.blacklight_configuration %>
+    <li><%= link_to t(:'spotlight.curation.sidebar.metadata'), spotlight.exhibit_edit_metadata_path(@exhibit) %></li>
     <li><%= link_to t(:'spotlight.curation.sidebar.search_facets'), spotlight.exhibit_edit_facets_path(@exhibit) %></li>
+    <li><%= link_to t(:'spotlight.curation.sidebar.sort_fields'), spotlight.exhibit_edit_sort_fields_path(@exhibit) %></li>
```

Comment:
I know these existed before but should all of these `@exhibit`s be `current_exhibit`?

For some reason I thought we got these in this file already....